src/BootstrapTable.js

```diff
@@ -872,6 +877,8 @@ class BootstrapTable extends Component {
   }
 
   _adjustHeight = () => {
+    if (typeof this.props.height === 'number' && !isNaN(this.props.height)) return this.props.height;
```

Comment:
Thanks for grabbing this feature request so fast!  I'm don't think this change works though.  Nothing uses the return value of `_adjustHeight` - instead, the method directly modifies the container height to be the requested height less the header height, unless the requested height is specified as a percentage.  (There's an existing bug here, in that it doesn't read `em` values, etc. correctly, but I don't think that's a priority.)  Would it be better to just do something like this?
```jsx
const stringHeight = String(this.props.height);
if (stringHeight.indexOf('%') === -1) {
  this.refs.body.refs.container.style.height = parseFloat(stringHeight, 10) - this.refs.header.refs.container.offsetHeight + 'px';
}
```